### ISSUE TITLE
Uninstall action doesn't uninstall game if rockstar launcher running

### ISSUE THREAD

**Issue by FriendsOfGalaxy** [tylerbrawl/Galaxy-Plugin-Rockstar#67]
When attempting to uninstall a game the play button changes to 'install' but the game does not actually get uninstalled if rockstar launcher is running in the background.

> 
> 2019-12-16 10:35:22,783 - galaxy.api.jsonrpc - INFO - Handling notification: method=uninstall_game, params={'game_id': '11'}
> 2019-12-16 10:35:22,783 - galaxy.task_manager - DEBUG - Task manager jsonrpc server: creating task 70 (uninstall_game)
> 2019-12-16 10:35:22,783 - galaxy.task_manager - DEBUG - Task manager plugin external: creating task 102 (uninstall_game)
> 2019-12-16 10:35:22,783 - root - DEBUG - ROCKSTAR_INSTALLER_PATH: ***
> 2019-12-16 10:35:22,784 - root - DEBUG - ROCKSTAR_UNINSTALL_REQUEST: Requesting to uninstall gta5...
> 2019-12-16 10:35:23,599 - galaxy.api.jsonrpc - INFO - Sending notification: method=local_game_status_changed, params={'local_game': LocalGame(game_id='11', local_game_state=<LocalGameState.None_: 0>)}
> 2019-12-16 10:35:23,599 - galaxy.api.jsonrpc - DEBUG - Sending 126 byte of data
> 2019-12-16 10:35:23,599 - galaxy.task_manager - DEBUG - Task manager plugin external: finished task 102 (uninstall_game)
> 2019-12-16 10:35:23,600 - galaxy.task_manager - DEBUG - Task manager jsonrpc server: finished task 70 (uninstall_game)
> 2019-12-16 10:35:23,874 - galaxy.api.jsonrpc - DEBUG - Received 92 bytes of data

> 
> 2019-12-16 10:48:08,454 - galaxy.api.jsonrpc - INFO - Handling notification: method=uninstall_game, params={'game_id': '10'}
> 2019-12-16 10:48:08,454 - galaxy.task_manager - DEBUG - Task manager jsonrpc server: creating task 32 (uninstall_game)
> 2019-12-16 10:48:08,455 - galaxy.task_manager - DEBUG - Task manager plugin external: creating task 48 (uninstall_game)
> 2019-12-16 10:48:08,455 - root - DEBUG - ROCKSTAR_INSTALLER_PATH: ***
> 2019-12-16 10:48:08,455 - root - DEBUG - ROCKSTAR_UNINSTALL_REQUEST: Requesting to uninstall mp3...
> 2019-12-16 10:48:10,765 - galaxy.api.jsonrpc - INFO - Sending notification: method=local_game_status_changed, params={'local_game': LocalGame(game_id='10', local_game_state=<LocalGameState.None_: 0>)}
> 2019-12-16 10:48:10,765 - galaxy.api.jsonrpc - DEBUG - Sending 126 byte of data
> 2019-12-16 10:48:10,765 - galaxy.task_manager - DEBUG - Task manager plugin external: finished task 48 (uninstall_game)
> 2019-12-16 10:48:10,765 - galaxy.task_manager - DEBUG - Task manager jsonrpc server: finished task 32 (uninstall_game)
> 2019-12-16 10:48:10,966 - galaxy.api.jsonrpc - DEBUG - Received 91 bytes of data

despite claiming that uninstalation finished for both game they are still playable from rockstar client and their files were not deleted.

If rockstar client was not running a proper uninstall prompt appears.

**Comment by tylerbrawl** [comment]
This is an interesting situation. One solution would be to cancel the uninstall request (i.e., do nothing) if the Rockstar Games Launcher is currently running, but that is not very intuitive. Another option would be to shut down the launcher and reboot it with the ```-uninstall``` parameter, but I am unsure of how well the launcher would handle this situation. Regardless, the status of the game should not be changed unless the game is uninstalled, so I will go ahead and fix that.

**Comment by tylerbrawl** [comment]
After some testing, it turns out that exiting the Rockstar Games Launcher will also shut down any game that is currently running. As such, there seems to be no safe solution to this problem. The safest thing to do is to ignore the request if the launcher is running, but that still is not a preferable option. If you or anybody else can think of a better solution, then let me know here.

**Comment by FriendsOfGalaxy** [comment]
I agree that it at least shouldn't change status in Galaxy in this case. Probably best to leave this issue open in case a proper solution comes up.

**Comment by tylerbrawl** [comment]
I will do so. The status no longer changes automatically, as well. (This was pushed to the ```fog_release``` branch as commit 12d445a114f45845b3c3de185d05796229eea888.)
